pytorch_lightning/callbacks/quantization.py

```diff
@@ -252,7 +252,7 @@ def _prepare_model(self, model: torch.nn.Module) -> None:
             model.qconfig = self._qconfig
 
         if self._check_feasible_fuse(model):
-            torch.quantization.fuse_modules(model, self._modules_to_fuse, inplace=True)
+            torch.ao.quantization.fuse_modules_qat(model, self._modules_to_fuse, inplace=True)
```

Comment:
AFAIK, `torch.ao` is available in PyTorch 1.10 or later, so it needs to be protected as we protect QConfig:
https://github.com/PyTorchLightning/pytorch-lightning/blob/cac02a04ba78a4bb69648f2918dbbf5798d5d84f/pytorch_lightning/callbacks/quantization.py#L32-L35

Is there a function that is equivalent to `torch.ao.quantization.fuse_modules_qat` in PyTorch 1.8 or 1.9?

Reply:
@ORippler ^^ :rabbit: 
we would need t make it back compatible to 1.8

Reply:
I looked up in the release notes, and for `pytorch<=1.10`, `torch.quantization.fuse_modules` implicitly uses the `Module.training` flag to determine whether fusion should happen in a QAT aware manner. I restored backwards compatibility by dynamic imports

Reply:
Thank you for your action!